dojo/tools/checkmarx/parser.py

```diff
@@ -110,7 +105,10 @@ def process_result_file_name_aggregated(self, dupes, findingdetail, query, resul
         titleStart = query.get('name').replace('_', ' ')
         description, lastPathnode = self.get_description_file_name_aggregated(query, result)
         sinkFilename = lastPathnode.find('FileName').text
-        title = "{} ({})".format(titleStart, ntpath.basename(sinkFilename))
+        if sinkFilename:
+            title = "{} ({})".format(titleStart, sinkFilename.split("/")[-1])
```

Comment:
What does happen, when a filename doesn't have a `/`? And does this work when the check is done on Windows (if this is possible)?

Reply:
the format is not dependent of the OS. the data in the report are always like this: `WebGoat/webgoat-lessons/missing-function-ac/src/main/java/org/owasp/webgoat/plugin/Users.java`.
This version is better as it doesn't fail with empty strings.

Reply:
ok